projects/ccd-case-ui-toolkit/src/lib/shared/pipes/link-cases-reason-code/ccd-link-cases-reason-code.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { LinkedCasesService } from '../../components/palette/linked-cases/services';
+
+@Pipe({
+  name: 'ccdLinkCasesReasonValue'
+})
+
```

Comment:
@johnbenjamin-hmcts - can remove empty line

Reply:
@Daniel-Lam I have updated the code now.